hudi-spark-datasource/hudi-spark-common/src/main/scala/org/apache/hudi/HoodieFileIndex.scala

```diff
@@ -284,8 +287,6 @@ case class HoodieFileIndex(spark: SparkSession,
         s"candidate file slices after data skipping: $candidateFileSliceSize; " +
         s"skipping percentage $skippingRatio")
 
-      hasPushedDownPartitionPredicates = true
```

Comment:
why removing this? This boolean tells `HoodiePruneFileSourcePartitions` rule to avoid repeated pruning.

Reply:
I move it to line 231.